.github/workflows/lint-test-build.yml

```diff
@@ -37,7 +41,19 @@ jobs:
         run: npm run build --if-present
 
       - name: Test
-        run: npm test
+        if: matrix.node-version == '22.x'
```

Comment:
Why testing only with nodejs v22?

Reply:
We test with both but we only put the 22 report into the PR. For the other one (20) the test results end up inside of the build log.

Reply:
So for v22 we run `npm run test:ci` and for 20 we run `npm run test`